pkg/handler/project/project.go

```diff
@@ -332,7 +332,7 @@ func (h *handler) GetMembers(c *gin.Context) {
 		return
 	}
 
-	heads, err := h.store.ProjectHead.GetByProjectID(h.repo.DB(), projectID)
+	heads, err := h.store.ProjectHead.GetActiveLeadsByProjectID(h.repo.DB(), projectID)
```

Comment:
why not include using `tx.DB()`

Reply:
we should use tx here to satisfy the consistent property. What happens if there is another tx which modifies this record when our tx still does not commit

Reply:
but the whole process has update inside. That why we should use tx here

Reply:
nvm, it on other function my bad